classes/shortcodes.php

```diff
@@ -91,8 +92,10 @@ function do_shortcode( $args ) {
 				'after'             => '',
 				'map_width'         => '75px',
 				'map_height'        => '75px',
-				'default_lat'       => get_post_meta( $post->ID, 'location_lat', true ),
-				'default_lng'       => get_post_meta( $post->ID, 'location_lng', true ),
+				'default_lat'       => get_post_meta( $post->ID, 'location_lat',
+					true ),
+				'default_lng'       => get_post_meta( $post->ID, 'location_lng',
+					true ),
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Multi-line function call not indented correctly; expected 16 spaces but found 20
* Closing parenthesis of a multi-line function call must be on a line by itself

<!-- Comment by [SideCI](https://sideci.com) -->